index.js

```diff
@@ -142,7 +142,7 @@ export default function typescriptProvider({negotiateProtocol}) {
 						],
 						ignoredByWatcherPatterns: [
 							...ignoredByWatcherPatterns,
-							...Object.values(relativeRewritePaths).map(to => `${to}**/*.js.map`),
+							...Object.entries(relativeRewritePaths).map(([to, from]) => `${compile === 'tsc' ? from : to}**`),
```

Comment:
Here too I think the entries are in the shape of `[from, to]`. And rather than ignoring all files in the directory we should only ignore those with the right extensions. Not sure right now if that should come from any configuration.

```suggestion
							...Object.entries(relativeRewritePaths).map(([from, to]) => `${compile === 'tsc' ? to : from}**/*.{ts,tsx}`),
```

Reply:
not sure about ignoring only some file extension, for two reasons:
- a change to a json file (as an example) might trigger tests before the compiler had the chance to run
- depending whether the **to** or **from** folder is ignored, the extensions list would change from ts-related to js-related

for the watcher I'd rather ignore the whole folders in order to:
- fully ignore the sources and only react when the compiler changed the output (auto compilation case)
- fully ignore what the compiler might change and only watch the sources (external compilation case)

is there some other scenario I'm not thinking of?

Reply:
That makes sense, but it changes the behavior compared to plain AVA. Changing a fixture file should cause tests to re-run.

But to get around that I think we'd need a way to block until compilation is complete?

Reply:
oh, didn't think about the fixtures, I haven't used them much until now. It makes sense not to change the standard behaviour.

I'll make the change to ignore only ts,tsx files (or something along those lines) for now 👍